Flow.Launcher.Core/Resource/Theme.cs

```diff
@@ -671,7 +671,15 @@ private void SetBlurForWindow(string theme, BackdropTypes backdropType)
                     windowBorderStyle.Setters.Remove(windowBorderStyle.Setters.OfType<Setter>().FirstOrDefault(x => x.Property.Name == "Background"));
                     windowBorderStyle.Setters.Add(new Setter(Border.BackgroundProperty, new SolidColorBrush(Colors.Transparent)));
                 }
-
+                
```

Comment:
Eliminated the issue where internal border values remained visible when the window border was forcibly set to a rectangle via Snap. This behavior only applies when blur is enabled; non-blur themes are unaffected.